### ISSUE TITLE
Create room button 

### ISSUE THREAD

**Issue by ghost** [mexili/incogly#37]
![Capture](https://user-images.githubusercontent.com/63122133/102063821-88cbdc00-3e1c-11eb-9677-b41f6606235e.PNG)

@sansyrox Sir, As shown in picture On phone "Create room" text is in the left side which is not looking good. I think shifting it into center will look good in the mobile phone. I want to work on this issue. Please assign it to me.

**Comment by sansyrox** [comment]
Hi @urvashi-code1255 ,

Thank you for the comment. But we are migrating to a new UI. So, I don't think that we should spend any time in fixing this one.

Will you be interested in implementing the new one?

**Comment by ghost** [comment]
Yes @sansyrox I want to contribute to this project's frontend :)

**Comment by sansyrox** [comment]
Will you like to propose a mobile UI for the newer front end here?

https://github.com/mexili/incogly/issues/38
